TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/autonomous/BaseAutonomousOpMode.java

```diff
@@ -22,9 +23,20 @@
     public DistanceSystem distanceSystem;
     public Marker markerSystem;
 
+    public double initPitch;
+    public double initRoll;
+
+    public int zone;
+    public int backCubeIn;
+    public int approachDeg0;
+    public int approachDeg1;
+    public int approachDeg2;
+    public double targDist1;
+
```

Comment:
backCubeIn? approachDeg0? Document please! (Just a line will do.)

Reply:
yeah. I will comment the meaning